### DISCUSSION TITLE
How to add a new property to vbtn?

### DISCUSSION THREAD

**Discussion by EpsilonYi** [vuetifyjs/vuetify#18996] [Q&A] [answered]
when i use tsx
`<VBtn onClick={handleClick}></VBtn>`

Property 'onClick' does not exist on type 'IntrinsicAttributes & { symbol?: any; replace?: boolean | undefined; flat?: boolean | undefined; exact?: boolean | undefined; block?: boolean | undefined; active?: boolean | undefined; ... 47 more ...; "v-slot:loader"?: false | ... 1 more ... | undefined; }'.
![image](https://github.com/vuetifyjs/vuetify/assets/16634792/89ccc3c7-80b4-4e81-9e31-80c5feee6492)

**Comment by cbrgpl** [comment]
Any news here? I use JSX in tests and don't understand how to listen for native events

**Reply by EpsilonYi** [reply, marked as answer]
add in vite-env.d.ts

`declare namespace JSX {
  interface IntrinsicClassAttributes {
    onClick?: FunctionConstructor
    id?: string
    key?: string
  }
}`
